Parser.php

```diff
@@ -86,8 +88,14 @@ private function content2Data($content)
                 $data['playlist'][$mediaSequence]['isDiscontinuity'] = true;
             }
 
-            if (preg_match('/^#EXTINF:(.+),/', $line, $matches)) {
+            if (preg_match('/^#EXTINF:(\d+),(.+)/', $line, $matches)) {
                 $data['playlist'][$mediaSequence]['duration'] = +$matches[1];
+                $data['playlist'][$mediaSequence]['name'] = $matches[2];
+                continue;
+            }
+
+            if (preg_match('/^#EXTGRP:(.+)/', $line, $matches)) {
```

Comment:
This is group of the channel: movie, music, history, and other. Many IPTV vendors support this tag.